web/auth/auth.go

```diff
@@ -1122,4 +1149,5 @@ func Routes(router *echo.Group) {
 	authorizeGroup.POST("/app", authorizeApp)
 
 	router.POST("/access_token", accessToken)
+	router.POST("/secret_exchange", secretExchange)
```

Comment:
Note for later: this route should be documented in `docs/auth.md`

Reply:
done